ddtrace/contrib/avro/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""
+The Avro integration will trace all Avro read / write calls made with the ``avro``
+library. This integration is enabled by default.
+
+Enabling
+~~~~~~~~
+
+The avro integration is enabled by default. Use
+:func:`patch()<ddtrace.patch>` to enable the integration::
+
+    from ddtrace import patch
+    patch(avro=True)
+
+Configuration
+~~~~~~~~~~~~~
+
+"""
+from ...internal.utils.importlib import require_modules
+
+
+required_modules = ["avro"]
+
+with require_modules(required_modules) as missing_modules:
+    if not missing_modules:
+        # Required to allow users to import from `ddtrace.contrib.avro.patch` directly
+        from . import patch as _  # noqa: F401, I001
```

Comment:
I'm actually wondering if we need to do this since it's a completely new integration 🤔 It would be consistent with what we have with the integrations we privatized, but I think we can start with just the necessary public methods with net new integrations now. Maybe @mabdinur has some thoughts?

Reply:
I believe you are correct, I have removed that patch file